packages/core/src/components/DataTable/TableHeader.tsx

```diff
@@ -142,5 +142,6 @@ export default withStyles(({ unit, color }) => ({
     justifyContent: 'space-between',
     paddingLeft: 2 * unit,
     paddingRight: 2 * unit,
+    borderBottom: `1px solid ${color.core.neutral[1]}`,
```

Comment:
`ui.border`?

Reply:
No :( `ui.border` is `color.neutral[2]`

Reply:
Any reason why this border is different?

Reply:
it'd be great if it were using the same border color as the rest of lunar